src/pages/settings/organization/Members.vue

```
@@ -1298,6 +1672,95 @@ async function handleInviteNewUserSubmit() {
     </Teleport>
 
+    <!-- Teleport for app access modal -->
+    <Teleport
+      v-if="dialogStore.showDialog && dialogStore.dialogOptions?.id === 'org-member-app-access'"
+      defer
+      to="#dialog-v2-content"
+    >
+      <div class="w-full">
+        <div class="max-h-[75vh] overflow-hidden">
+          <div class="grid h-full gap-6 lg:grid-cols-[minmax(0,1fr)_minmax(0,1fr)]">
+          <div>
```

ESLint

[error] Expected indentation of 12 spaces but found 10 spaces.

(vue/html-indent)